### PR TITLE
Multi port updated

### PR DESCRIPTION
**Pull request by olbjo**
Misc updates to pass Automated RT test suite on Raspberry Pi with an usb/ethernet dongle as a second port.

Includes / rebased on #319

### CODE REVIEW

**Review by pyhys — COMMENTED** [review]
Looks great!
My only comment is that maybe we should change the GSDML filename (date part) when modifying the file.

**Review by pyhys — APPROVED** [review]

File: src/device/pf_port.c [hunk 1]
```diff
@@ -101,13 +101,13 @@ const pnet_port_cfg_t * pf_port_get_config (pnet_t * net, int loc_port_num)
 uint16_t pf_port_loc_port_num_to_dap_subslot (int loc_port_num)
```
Comment (hefloryd) [inline src/device/pf_port.c:101]: These functions should have tests (or regression tests for the problem being fixed).

Comment (olbjo) [reply src/device/pf_port.c:101]: Tests added.
Done

File: sample_app/GSDML-V2.4-rtlabs-IODevice-20201211.xml [hunk 2]
```diff
@@ -56,15 +56,25 @@
                   </InterfaceSubmoduleItem>
                   <PortSubmoduleItem ID="IDS_P1" SubmoduleIdentNumber="0x00008001" SubslotNumber="32769" TextId="IDT_NAME_PS1" MaxPortRxDelay="350" MaxPortTxDelay="160">
                      <MAUTypeList>
+<!-- 
+MAUTypeItems shall match the actual network interfaces of the device.
+Current list is works for Raspberry Pi, Linksys usb/ethernet dongle and xmc sample targets
```
[the remark this answers is not in the document]
Comment (olbjo) [reply sample_app/GSDML-V2.4-rtlabs-IODevice-20201211.xml]: Done

File: doc/multiple_ports.rst [hunk 3]
```diff
@@ -96,11 +96,16 @@ Run ``ifconfig`` to check that the bridge is up and its network interfaces are a
 Configuration of p-net stack and sample application 
 ---------------------------------------------------------
 To run p-net and the sample application with multiple ports a couple
-of things need to be done:
+of things need to be done. Note that the settings described in the
```
[the remark this answers is not in the document]
Comment (olbjo) [reply doc/multiple_ports.rst:99]: Done

**Pull request merged**